src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,22 @@
+package racingcar;
+
+public class Car {
+
+    private int position = 0;
+
+    public void move(int distance) {
+        if (judgeForth(distance)) position++;
+    }
+
+    public boolean judgeForth(int degree) {
+        if (degree >= 4 && degree <= 9) {
```

Comment:
해당 4, 9는 상수화를 하면 어덜까요?
아울러 
```suggestion
        reutrn degree >= 4 && degree <= 9
```
이런식으로 사용해도 동일할거 같아요

이런 방식이 더 보기 좋은거 같아요~